Nef_3/include/CGAL/Nef_3/polygon_mesh_to_nef_3.h

```diff
@@ -192,16 +192,21 @@ void polygon_mesh_to_nef_3(PolygonMesh& P, SNC_structure& S, FaceIndexMap fimap,
   typedef Halfedge_around_target_circulator<PolygonMesh>
                                Halfedge_around_vertex_const_circulator;
 
+  struct Face_info
+  {
+    Vector_3 normal;
+    CGAL_assertion_code(std::size_t num_edges);
+  };
```

Comment:
Are should sure this will compile with disabled assertions?

Reply:
@lrineau yeah. CGAL_assertion_code is defined as `#define CGAL_assertion_code(CODE)` when disabled and `#define CGAL_assertion_code(CODE) CODE` when enabled. Furthermore, the c++ spec allows for an additional optional comma at the end of an initialiser list... who knew.

https://en.cppreference.com/w/c/language/initialization

Reply:
Oh, and the additional `;` is also allowed in the struct ;)

Reply:
http://cpp.sh/8l4tm

Reply:
> Oh, and the additional `;` is also allowed in the struct ;)

You are right, but there is a warning with `-pedantic`.